ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -191,6 +193,11 @@ public class RNTrackPlayer: RCTEventEmitter, AudioPlayerDelegate {
             self?.sendEvent(withName: "remote-play", body: nil)
             return MPRemoteCommandHandlerStatus.success
         }
+
+        player.remoteCommandController.handleTogglePlayPauseCommand = { [weak self] _ in
```

Comment:
@anderslemke search for: `player.remoteCommandController.handleTogglePlayPauseCommand` -> this method already exists.. does it not work as expected?

Reply:
Yeah it seems that it exists but wasn't used until now, corret?

Reply:
Yes. That already exists. I'll remove this.

I don't think it was used before though, as @appinteractive points out.